src/electron-main.js

```diff
@@ -263,7 +263,7 @@ ipcMain.on('ipcCall', async function(ev, payload) {
             store.set('minimizeToTray', args[0]);
             break;
         case 'getAutoHideMenuBarEnabled':
-            ret = global.mainWindow.isMenuBarAutoHide();
+            ret = global.mainWindow.autoHideMenuBar;
```

Comment:
Is this part safe to land with 8.0.1 and earlier, or does it require 8.0.3? (If safe, maybe land it separately from the version change to avoid SSO complication above?)

Reply:
Safe to land on any 7+ but also not needed until 9.